setup.py

```diff
@@ -47,6 +60,9 @@
     if not on_rtd:
         # Do not add TF if we are installing GPflow on readthedocs
         requirements.append(tf_cpu)
+        gast_requirement = 'gast>=0.2.2,<0.3' if parse_version(latest_version('tensorflow')) < parse_version('2.2') else 'gast>=0.3.3'
```

Comment:
```suggestion
        gast_requirement = 'gast>=0.2.2,<0.3' if latest_version('tensorflow') < parse_version('2.2') else 'gast>=0.3.3'
```
and following the previous suggestions it's already a Version object so no need to parse it anymore!